types/ramda/ramda-tests.ts

```diff
@@ -474,7 +474,6 @@ type Pair = KeyValuePair<string, number>;
 };
 
 () => {
-    const x          = R.prop("x");
```

Comment:
What was this doing? anything?

Reply:
Looks like nothing. Probably initially used in the 4 lines that come after it, but it doesn't seem to do or assert anything here, other than that `R.prop()` *can* take a string and doesn't return `void` or `never`.